packages/foreman/foreman/foreman.spec

```diff
@@ -1005,18 +1005,20 @@ fi
 exit 0
 
 %post service
-%systemd_post %{name}.socket
```

Comment:
This is incorrect. It needs to be there and sets up the initial installation. See https://github.com/systemd/systemd/blob/6b222c4b0227f9914446ac54754aea867f742be4/src/core/macros.systemd.in#L46-L52

I do think `%{?*}` in there means you can combine them to be done in parallel.